station.go

```diff
@@ -614,3 +631,49 @@ func (sd *schemaDetails) validateGraphQlMsg(msg any) ([]byte, error) {
 	}
 	return msgBytes, nil
 }
+
+func (sd *schemaDetails) validAvroSchemaMsg(msg any) ([]byte, error) {	
+	var (
+		msgBytes []byte
+		err      error
+		message  interface{}
+	)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	switch msg.(type) {
+	case []byte:
+		msgBytes = msg.([]byte)
+		if err := json.Unmarshal(msgBytes, &message); err != nil {
+			err = errors.New("Bad Avro format - " + err.Error())
+			return nil, memphisError(err)
+		}
+		
+	default:
```

Comment:
In protobuf we allow users to send map[string]interface{} in order people won't have to hole the proto file locally, is it possible here as well?

Reply:
Yes. I've added it now.